### PR TITLE
Sweet overlay.

### PR DESCRIPTION
**Pull request by dconnolly**
videojs-overlay didn't let me show one at init, and does the show/hide
by creating/removing elements from the dom. This is bad when loading a
third party badge image, which has a small lag when loading fresh.  Now
just doing overlay manually and show/hiding it with `display: none`
instead.

output

### CODE REVIEW

File: public/css/components/sections/_main-preview.scss [hunk 1]
```diff
@@ -23,3 +23,48 @@
...
+   top: 37.5%;
+   left: 0;
+   width: 100%;
+   margin: 0;
+   padding: 0;
+   background-color: rgba(250, 252, 255, 0.95);
+   color: #757A7D;
+   font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+   .videojs-hero {
```
Comment (mmcc) [inline public/css/components/sections/_main-preview.scss]: We haven't been good about putting together an official SCSS style guide, but we've tried to keep nesting for just pseudo-elements. You don't have to worry about it in in this PR, we can take another pass after it's pulled in.

**Comment by mmcc** [comment]
@dconnolly Looks good! I'd like to go ahead and pull this in, but looks like we need a rebase. Mind rebasing from the current 5.0 branch?

**Comment by dconnolly** [comment]
@mmcc rebased/fixed merge conflicts, cleaned up and refactored a few things due to plugins view merge. Have another look?

**Comment by mmcc** [comment]
@dconnolly I haven't been very careful about squashing so far, so... :rocket:

**Comment by dconnolly** [comment]
Cheers!

**Pull request merged**